ruby/ql/lib/codeql/ruby/security/InsecureDownloadQuery.qll

```diff
@@ -0,0 +1,31 @@
+/**
+ * Provides a taint tracking configuration for reasoning about download of sensitive file through insecure connection.
```

Comment:
In this case it looks like we're using plain dataflow rather than taint tracking. I'm guessing that taint tracking might be too noisy for this query?

Reply:
It's a good question, I didn't think much about this when porting the query over from JS but I think you're right, we'd probably not recognise a lot of sanitisation guards and generate FPs. But maybe we should consider trying out taint tracking if we ever want to boost the sensitivity of this query.